modules/site-health/audit-full-page-cache/load.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Module Name: Audit Full Page Cache
+ * Description: Adds a Full page Cache checker in Site Health checks.
```

Comment:
Let's align this more with the other Site Health module descriptions.
```suggestion
 * Description: Adds a check for full page cache in Site Health status.
```